packages/lodestar/src/eth1/impl/ethers.ts

```diff
@@ -132,30 +140,38 @@ export class EthersEth1Notifier implements IEth1Notifier {
    * @param toNumber
    */
   public async processBlocks(toNumber: number): Promise<void> {
+    this.logger.info(`Processing eth1 blocks from ${this.lastProcessedEth1BlockNumber + 1} to ${toNumber}`);
     let rangeBlockNumber = this.lastProcessedEth1BlockNumber;
     while (rangeBlockNumber < toNumber && this.startedProcessEth1) {
-      const blockNumber = Math.min(this.lastProcessedEth1BlockNumber + 100, toNumber);
+      const endRangeBlockNumber = Math.min(this.lastProcessedEth1BlockNumber + 1000, toNumber);
```

Comment:
I has to tweak this a couple of times to find a best number. Do we want to make this as an eth1 option?